public/data/productListData-doyoung.json

```
@@ -0,0 +1,51 @@
+[
+  {
+    "id": 1,
+    "productName": "(남성)프렌치 레귤러핏 반팔 폴로",
+    "productColors": "+ 1 clours ",
+    "productPic": "https://imageapac1.lacoste.com/dw/image/v2/BBCL_PRD/on/demandware.static/-/Sites-master/default/dw43a87826/PH731E-51N_031_20.jpg?imwidth=690&impolicy=product",
+    "productPrice": "10,000원"
```

```suggestion
    "productPrice": 10000
```
제 생각에는 price는 number로 만들어둬야 추후 값을 계산하거나 할 때 수월할 것 같습니다. 혹시 백엔드에서 값을 이렇게 넘겨주기로 했나요? 키값도 이상해서 같이 논의 필요할 거 같습니다.

---

price 형태 number 로 수정하였습니다. 임시 mock-data라 키값 등 구체적인 데이터 형태는 B/E와 논의 예정입니다.